docs/docs/reference/resources/experiment/index.html

```diff
@@ -1,9 +1,9 @@
+apiVersion: pipelines.kubeflow.org/v1alpha6 kind: Experiment metadata: name: penguin-experiment spec: description: 'An experiment for the penguin pipeline' Fields Name Description spec.description The description of the experiment. "><meta property="og:url" content="https://sky-uk.github.io/kfp-operator/docs/reference/resources/experiment/"><meta property="og:site_name" content="KFP-Operator"><meta property="og:title" content="Experiment"><meta property="og:description" content="The Experiment resource represents the lifecycle of Experiments, and can be configured as follows:
```

Comment:
is this file autogenerated? How come there's no indenting

Reply:
yes, anything in `docs/` is generated from `docs-gen/` and served with github pages